MidasFrontend/feArduino.cxx

```diff
@@ -34,6 +34,11 @@ Arduino frontend for mPMT test stand.
 #define ODB_VAR_DESTINATION  ODB_PATH_SETTINGS "/Destination"
 #define ODB_VAR_VELOCITY     ODB_PATH_SETTINGS "/Velocity"
 
+// Note: these must be exactly 4 letters long
+#define ODB_BANK_GANTRY      "GANT"
+#define ODB_BANK_STATUS      "STAT"
```

Comment:
Can you remind me what STAT tells us? Was it a flag that indicates the system is moving?

Reply:
It currently only stores the result of `GET_STATUS` (either moving, homing, idle etc.). Could potentially throw other stuff in there as well in the future. Might throw the limit switch statuses in.